<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="durationlabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>pos/tot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>980</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border: 2px solid grey;
border-radius: 5px;
}

QProgressBar::chunk {
background-color: rgb(37, 86, 201);
/*background-color: #05B8CC;*/
width: 20px;
/*margin: 0.5px;*/
}

</string>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="appNameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>510</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nexell Video Player</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playListButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>playlist</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumelabel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>310</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Vol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="volumeProgressBar">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>41</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border: 2px solid grey;
border-radius: 5px;
}

QProgressBar::chunk {
background-color: rgb(37, 86, 201);
/*background-color: #05B8CC;*/
height: 20px;
/*margin: 0.5px;*/
}
</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="subTitleLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>510</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>510</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speedButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>350</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x 1</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>durationlabel</zorder>
   <zorder>closeButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>appNameLabel</zorder>
   <zorder>playListButton</zorder>
   <zorder>volumelabel</zorder>
   <zorder>prevButton</zorder>
   <zorder>playButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>nextButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>volumeProgressBar</zorder>
   <zorder>subTitleLabel</zorder>
   <zorder>speedButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
